README.md

```diff
@@ -111,6 +111,14 @@ This command specifies the document's IPAC lead author:
 \ipaclead[email=galileo@example.com]{Galileo Galilei}
 ```
 
+#### \interfacepartner
+
+This command specifies the document's interface partner (for SSDC-IF documents):
+
+```tex
+\interfacepartner[email=galileo@example.com]{Galileo Galilei}
```

Comment:
This will just be the name of a SPHEREx subsystem, not a person, so no email is needed.